packages/ui/components/avatar/AvatarGroup.tsx

```diff
@@ -16,48 +16,45 @@ export type AvatarGroupProps = {
 };
 
 export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
-  const avatars = props.items.slice(0, 4);
   const LENGTH = props.items.length;
   const truncateAfter = props.truncateAfter || 4;
+  /**
+   * First, filter all the avatars object that have image
+   * Then, slice it until before `truncateAfter` index
+   */
+  const displayedAvatars = props.items.filter((avatar) => avatar.image).slice(0, truncateAfter);
+  const numTruncatedAvatars = LENGTH - displayedAvatars.length;
 
   return (
     <ul className={classNames("flex items-center", props.className)}>
-      {avatars.map((item, enumerator) => {
-        if (item.image != null) {
-          if (LENGTH > truncateAfter && enumerator === truncateAfter - 1) {
-            return (
-              <li key={enumerator} className="relative -mr-[4px] inline-block ">
-                <div className="relative">
-                  <div className="h-90 relative min-w-full scale-105 transform border-gray-200 ">
-                    <Avatar className="" imageSrc={item.image} alt={item.alt || ""} size={props.size} />
-                  </div>
-                </div>
-                <div
-                  className={classNames(
-                    "absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 text-white",
-                    props.size === "sm" ? "text-base" : "text-2xl"
-                  )}>
-                  <span>+{LENGTH - truncateAfter - 1}</span>
-                </div>
-              </li>
-            );
-          }
-          // Always display the first Four items items
-          return (
-            <li key={enumerator} className="-mr-[4px] inline-block">
-              <Avatar
-                className="border-gray-200"
-                imageSrc={item.image}
-                title={item.title}
-                alt={item.alt || ""}
-                accepted={props.accepted}
-                size={props.size}
-                href={item.href}
-              />
-            </li>
-          );
-        }
-      })}
+      {displayedAvatars.map((item, idx) => (
+        <li key={idx} className="-mr-[4px] inline-block">
+          <Avatar
+            className="border-gray-200"
+            imageSrc={item.image}
+            title={item.title}
+            alt={item.alt || ""}
+            accepted={props.accepted}
+            size={props.size}
+            href={item.href}
+          />
+        </li>
+      ))}
+      {numTruncatedAvatars > 0 && (
+        <li
+          className={classNames(
+            "relative -mr-[4px] mb-1 inline-flex justify-center overflow-hidden rounded-full bg-darkgray-300",
+            props.size === "sm" ? "min-w-6 h-6" : "min-w-16 h-16"
+          )}>
```

Comment:
after seeing bg-black in action i decided to change it to bg-darkgray-300

Reply:
@PeerRich I saw that the lint workflow failed due to the classname order rule, so I pushed a commit to fix that